src/org/rascalmpl/library/Prelude.java

```diff
@@ -1149,6 +1149,16 @@ public static String consumeInputStream(Reader in) throws IOException {
 		}
 		return res.toString();
 	}
+
+	public static byte[] consumeInputStream(InputStream in) throws IOException {
```

Comment:
might be some places in Prelude we can replace with this?

Reply:
We did that before; those were all `Readers` I  believe.